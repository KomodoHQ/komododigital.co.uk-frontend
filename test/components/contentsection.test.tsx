import React from 'react';
import ContentSection from '../../src/components/contentsection';
import renderer from 'react-test-renderer';

test('ContentSection element is rendered', () => {
  const component = renderer.create(
    <ContentSection title={'Test'} subtitle={'Testy'}>
      <p>Test</p>
    </ContentSection>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
