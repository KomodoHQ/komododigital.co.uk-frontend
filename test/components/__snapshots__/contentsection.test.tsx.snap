// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentSection element is rendered 1`] = `
<div
  className="komodoGridWrapper cs-wrapper  "
  style={
    Object {
      "background": "none",
    }
  }
>
  <div
    className="Content-Section"
  >
    <span>
      Testy
    </span>
    <h2>
      Test
    </h2>
    <div>
      <p>
        Test
      </p>
    </div>
  </div>
</div>
`;
